tests/test_checkers.py

```diff
@@ -995,6 +998,59 @@ def test_text_real_file(self, tmp_path: Path):
             check_type(f, TextIO)
 
 
+class TestIntersectingProtocol:
+    SIT = TypeVar("SIT", bound=object, covariant=True)
+
+    class SizedIterable(
+        Sized,
+        Iterable[SIT],
+        Protocol[SIT],
+    ): ...
+
+    @pytest.mark.parametrize(
+        ("subject", "predicate_type"),
```

Comment:
Follow the style of existing parametrization calls.
```suggestion
        "subject, predicate_type",
```